crates/hotshot/src/lib.rs

```diff
@@ -305,6 +305,10 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>> SystemContext<TYPES, I> {
     pub async fn start_consensus(&self) {
         #[cfg(feature = "dependncy-tasks")]
         error!("HotShot is running with the dependency tasks feature enabled!!");
+
+        #[cfg(all(feature = "rewind", not(debug_assertions)))]
```

Comment:
Thoughts?

Reply:
Does a `compile_error!` work here?

Reply:
oooh yes it does! Great suggestion. Done.